### PR TITLE
Added Windows compatible version of build:debug script

### PR DESCRIPTION
**Pull request by simihartstein**

### CODE REVIEW

File: syncToTestApp.ps1 [hunk 1]
```diff
@@ -0,0 +1,38 @@
+
+if( $args[0] -ne $null ) {
```
Comment (toddsouthenbentley) [inline syncToTestApp.ps1:2]: I think we should be consistent on the spaces before and after parens. Personally, I'd prefer a space after `if` and not spaces inside the parens if that's OK in Powershell.

Comment (simihartstein) [reply syncToTestApp.ps1:2]: sure. I don't think there's a convention for powershell, but it's syntactically correct, so I'll update it.

**Review by toddsouthenbentley — APPROVED** [review]

**Review by toddsouthenbentley — APPROVED** [review]

**Comment by tcobbs-bentley** [comment]
@simihartstein The CLA error above is very strange. When I hover over your username, it shows you having an account, and yet the error says that the email you used with this commit isn't a GitHub user at all. Any idea what might be going wrong? Also, have you accepted the CLA?

**Comment by simihartstein** [comment]
fixing my email. Will resubmit

**Pull request closed without merging**